src/app/zap-templates/common/ClusterTestGeneration.js

```diff
@@ -826,6 +826,28 @@ function isLiteralNull(value, options)
   return (value === null) || (value instanceof NullObject);
 }
 
+function isHexString(value)
+{
+  return value && value.startsWith('hex:');
+}
+
+function octetStringFromHexString(value)
+{
+  const hexString = value.substring(4);
```

Comment:
Might be less fragile to have a `kHexPrefix` string, and use `kHexPrefix.length` here and in the next function.